langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -38,6 +38,10 @@ type AnydataType anydata;
 # It corresponds to the Clone(v) abstract operation,
 # defined in the Ballerina Language Specification.
 #
+# ```
+# [1, 2, 3, 4].clone() ⇒ [1,2,3,4]
```

Comment:
Thinking out loud, does this actually convey what's happening? May make sense to have a variable here, right?

```suggestion
# int[] arr = [1, 2, 3, 4];
# int[] clone = [1, 2, 3, 4].clone();
# clone ⇒ [1,2,3,4]
# arr === clone ⇒ false
```